src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds

```diff
@@ -37,5 +37,24 @@
     </Project>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(__BuildType)' == 'Release'">
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroName)' == 'rhel'" Include="rhel/Microsoft.NETCore.Runtime.CoreCLR-symbols.pkgproj">
```

Comment:
I would suggest that the name end with ".Symbols".

@weshaggard @ericstj Are there conventions for naming packages?